scripts/build-website.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('glob');
+const { execSync } = require('child_process');
+
+const websitePath = path.join(__dirname, '..', 'website');
+const websiteDistPath = path.join(websitePath, 'dist');
+const examplesSourcePath = path.join(websitePath, 'examples');
+const examplesDistPath = path.join(websiteDistPath, 'examples');
+
+// const examples = glob.sync(path.join(sourcePath, '*'));
```

Comment:
```suggestion
// const examples = glob.sync(path.join(examplesSourcePath, '*'));
```

Reply:
Good catch, but it will be updated later in any cases.